src/box/lua/stat.c

```diff
@@ -67,6 +67,18 @@ inject_current_stat(struct lua_State *L, const char *name, size_t val)
 	lua_pop(L, 1);
 }
 
+static void
+iproto_stream_queue_stats_inject(struct lua_State *L, struct iproto_stats *s)
```

Comment:
Can you merge all functions "injecting" current values for all stats?

Reply:
Fixed introduce function inject_iproto_stats